numpyro/distributions/continuous.py

```diff
@@ -1044,6 +1044,109 @@ def infer_shapes(
         return batch_shape, event_shape
 
 
+class MultivariateStudentT(Distribution):
+    arg_constraints = {
+        "df": constraints.positive,
+        "loc": constraints.real_vector,
+        "scale_tril": constraints.lower_cholesky,
+    }
+    support = constraints.real_vector
+    reparametrized_params = ["df", "loc", "scale_tril"]
+
+    def __init__(
+        self,
+        df,
+        loc=0.0,
+        scale_tril=None,
+        validate_args=None,
+    ):
+        if jnp.ndim(loc) == 0:
+            (loc,) = promote_shapes(loc, shape=(1,))
+        batch_shape = lax.broadcast_shapes(
+            jnp.shape(df), jnp.shape(loc)[:-1], jnp.shape(scale_tril)[:-2]
+        )
+        (self.df,) = promote_shapes(jnp.array(df), shape=batch_shape)
```

Comment:
I think we don't need to use `jnp.array` here (just to be more consistent with other distributions).